pkg/operator/ceph/controller/spec.go

```diff
@@ -798,3 +800,14 @@ func ConfigureExternalMetricsEndpoint(ctx *clusterd.Context, monitoringSpec ceph
 func extractMgrIP(rawActiveAddr string) string {
 	return strings.Split(rawActiveAddr, ":")[0]
 }
+
+func GetContainerImagePullPolicy(containerImagePullPolicy string) *v1.PullPolicy {
```

Comment:
How about always returning the value instead of a pointer? Then you don't have to dereference it everywhere else.
```suggestion
func GetContainerImagePullPolicy(containerImagePullPolicy string) v1.PullPolicy {
```